src/main/java/com/spotinst/sdkjava/model/bl/admin/account/BlAccountAdmin.java

```diff
@@ -86,16 +106,26 @@ public Builder setOrganizationId(final String organizationId) {
             return this;
         }
 
+        public Builder setCloudProvider(final String cloudProvider) {
+            account.setCloudProvider(cloudProvider);
+            return this;
+        }
+
+        public Builder setProviderExternalId(final String providerExternalId) {
+            account.setProviderExternalId(providerExternalId);
+            return this;
+        }
+
         public BlAccountAdmin build() {
             return account;
         }
     }
     //endregion
 
     //region isSet methods
-    // Is id Set boolean method
+    // Is accountId Set boolean method
     @JsonIgnore
-    public boolean isIdSet() { return isSet.contains("id"); }
+    public boolean isAccountIdSet() { return isSet.contains("accountId"); }
```

Comment:
separate it  to new lines

Reply:
Formatted all methods throughout this file.